src/chk/chk_leader.c

```diff
@@ -1727,6 +1727,25 @@ chk_leader_pool_mbs(struct chk_sched_args *csa)
 	return rc;
 }
 
+/*
+ * Whether need to stop current check instance or not.
+ *
+ * \return	1:	the check is completed.
+ * \return	0:	someone wants to stop the check.
+ * \return	-1:	continue the check.
+ */
```

Comment:
[minor] return number is confusing, 0 means false, which means not stopped.
I think it might be improved like:

0:  continue the check
1: check is completed and need be stopped.
2:  someone wants to stop the checker.

then in the caller, you could:
if check_needer_need_stop() > 0
      xxxx

Reply:
That is mainly because: for check leader scheduler, positive exit status means the check is completed, zero means stop explicitly, then we can directly use chk_leader_need_stop() return value without other conversion.